rfcs/0000-rfc-process/text.md

```diff
@@ -37,10 +37,11 @@ The following are the steps to writing and approving a new RFC (not modifying):
 2. Create a new directory under the `rfcs` directory named `XXXX-name-of-rfc`, where `name-of-rfc` is the name of your RFC, not those words directly.
 3. Copy over the contents of the `templates` directory into your newly created directory. This contains a folder for images and the RFC template.
 4. Rename `rfc_template.md` to `text.md` and edit it. Add images that you'd like to include in the `imgs` folder.
-5. When you are ready for review, create a pull request. Add your approvers (and any other people you'd like to review your work) as reviewers on the PR. Please note that [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com) and [Sara Gerber](mailto:sara.gerber@chanzuckerberg.com) are required Reviewers for all RFCs so that CZI Central Infra and CZI Security teams are aware of cellxgene technical work. They may be removed as reviewers if the RFC is non-technical.
-6. As a courtesy, post a heads up on the [#org-sci-eng-single-cell](https://chanzuckerbergteam.slack.com/archives/GQGPP7925) Slack channel that there is a design available for review, specifically calling out the approvers. Generally, two weeks is nice for substantial designs to get reviewed, but use your best judgement on how much time you'd like for your design to be under review.
-7. Once all comments are addressed and the _approvers_ have approved your pull request, **change the name of your directory** from `XXXX-name-of-rfc` to pick the next number (for example, this one will be `0000-rfc-process`).
-8. Merge your PR! Delete your branch! Close your design ticket! Code away!
+5. When you are ready for review, create a pull request. Add your approvers (and any other people you'd like to review your work) as reviewers on the PR. Please note that [Girish Patangay](mailto:girish.patangay@chanzuckerberg.com) and [David Nichols](mailto:dnichols@chanzuckerberg.com) are required Reviewers for all RFCs so that CZI Central Infra and CZI Security teams are aware of cellxgene technical work. They may be removed as reviewers if the RFC is non-technical.
```

Comment:
Should we encourage design owner to schedule workshop with key collaborators early in the design process to brainstorm and create a game plan before the first draft of RFC is written? And also optionally create a temp Slack channel to allow focused communication and design updates

My impression is that the feedback loop is long now because collaborators' feedback only start coming in after RFC is written, which seems late 

Ideally it almost feels like the main architecture of an RFC shouldn't be a surprise to any key collaborators/reviewers, because they were involved in the design from the beginning. And then other non-key collaborators/reviewers can use RFC PR review time to provide extra feedback where needed

Would love to see what people think!

Reply:
This will result in fewer major edits in the first draft.

Reply:
That sounds like a good idea! I figured that ultimately it is up to the person(s) writing the RFC to drive the design to completion whether it be through any or all of the above means, but I'll note them down as recommendations! Thanks!